python/pyspark/errors/error-conditions.json

```diff
@@ -1002,6 +1002,73 @@
       "Cannot start a remote Spark session because there is a regular Spark session already running."
     ]
   },
+  "SESSION_MUTATION_IN_DECLARATIVE_PIPELINE": {
+    "message": [
+      "Session mutation <method> is not allowed in declarative pipelines."
+    ],
+    "sub_class": {
+      "RUNTIME_CONF_SET": {
```

Comment:
Nitpick: should the `SET` be on the other side of `RUNTIME_CONF` for consistency?

Reply:
good catch!